packages/framework/src/Resources/views/Admin/Grid/Grid.html.twig

```diff
@@ -363,7 +363,11 @@
 {% endblock %}
 
 {% block grid_value_cell_type_Shopsys_FrameworkBundle_Component_Money_Money %}
-    {{ value|priceWithCurrencyAdmin }}
+    {% if 'domainId' in row|keys %}
```

Comment:
That's a really clever solution 👍 

Can you please document the behavior in `grid.md` or `grid-rendering-customization.md`?